rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -598,6 +602,11 @@ message Proposal {
     //
     // Id = 15.
     AdvanceSnsTargetVersion advance_sns_target_version = 19;
+
+    // SetCustomProposalTopics
+    //
+    // Id = 16;
+    SetCustomProposalTopics set_custom_proposal_topics = 20;
```

Comment:
Maybe this should be named "SetTopicsForCustomProposals"

Reply:
I thought about it, and realized that I still prefer the more concise name for the proposal, as I don't think the opportunity for misunderstanding its purpose is too high (since topics are fixed for all SNSs).